src/Tribe/Aggregator/Settings.php

```diff
@@ -187,7 +187,7 @@ public function is_ea_authorized_for_eb() {
 	 * @param WP_Screen $screen The current screen instance.
 	 */
 	public function maybe_clear_meetup_credentials( $screen ) {
-		if ( 'tribe_events_page_tribe-common' !== $screen->base ) {
+		if ( 'tribe_events_page_tec-events-settings' !== $screen->base ) {
```

Comment:
Just added a couple methods in common and others on the TEC Admin\Settings class so that we can get this without hardcoding it: https://github.com/the-events-calendar/the-events-calendar/pull/3880/files#diff-e4d0aa550c265b7f74fd03910dfff06ef7ae724f65875f8d05f9e4d48badf217R159